PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalClientUnitTest.kt

```diff
@@ -182,7 +183,7 @@ class PayPalClientUnitTest {
     }
 
     @Test
-    fun createPaymentAuthRequest_whenLaunchesBrowserSwitchAsNewTaskEnabled_setsNewTaskOption() {
+    fun createPaymentAuthRequest_launchesBrowserSwitchWith() {
```

Comment:
Is this missing the rest of the test name?

Reply:
Good catch! Just pushed an update.